lib/start.js

```diff
@@ -80,14 +80,20 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
     if (user_data_dir) {
       // clear the data directory before doing a network test
       fs.removeSync(user_data_dir.replace('\\', ''))
+      if (fs.existsSync(networkLogFile)) {
```

Comment:
Deleting the files before starting the audit helps avoid stale results if the new file is not created.